engine/src/gradient.cpp

```diff
@@ -352,7 +352,7 @@ static void MCGradientFillFetchProperty(MCExecContext& ctxt, MCGradientFill* p_g
         case P_GRADIENT_FILL_RAMP:
         {
             MCAutoStringRef t_data;
-            MCGradientFillRampUnparse(p_gradient->ramp,p_gradient->ramp_length, &t_data);
+            /* UNCHECKED */ MCGradientFillRampUnparse(p_gradient->ramp,p_gradient->ramp_length, &t_data);
```

Comment:
I think that if this fails, it might be a good idea to not set `r_value` at all.